fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/batch/NereidsRewriteJobExecutor.java

```diff
@@ -90,6 +92,11 @@ public NereidsRewriteJobExecutor(CascadesContext cascadesContext) {
                 .add(bottomUpBatch(ImmutableList.of(new AdjustAggregateNullableForEmptySet())))
                 .add(topDownBatch(ImmutableList.of(new EliminateGroupByConstant())))
                 .add(topDownBatch(ImmutableList.of(new NormalizeAggregate())))
+                .add(topDownBatch(ImmutableList.of(new ExtractAndNormalizeWindowExpression())))
+                //execute NormalizeAggregate() again to resolve nested AggregateFunctions in WindowExpression,
+                //e.g. sum(sum(c1)) over(partition by avg(c1))
+                .add(topDownBatch(ImmutableList.of(new NormalizeAggregate())))
```

Comment:
it is weird, i think after normalize aggregate we should get
```
project(sum(a1) over (partition by a2))
+---agg(sum(c1) as a1, avg(c1) as c2)
```
and then we do extract window expression